src/main/java/cloud/orbit/spring/OrbitActorsProperties.java

```diff
@@ -28,19 +28,33 @@
 
 package cloud.orbit.spring;
 
-import cloud.orbit.actors.Stage;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+import cloud.orbit.actors.Stage;
+import cloud.orbit.actors.extensions.ActorExtension;
+
 import java.util.List;
 
 @ConfigurationProperties(prefix = "orbit.actors")
 public class OrbitActorsProperties {
+    private String basePackages;
     private String clusterName;
     private String nodeName;
     private Stage.StageMode stageMode;
+    private List<ActorExtension> extensions;
```

Comment:
Yes... I would say this is not something orbit-spring should support. Spring has its own way to do that type of instantiation and adding in orbit-hk2 understandings into a Spring project doesn't align with how Spring developers would be looking to use the library.